test/unit/composable_component/controller/leftHeader/ControllerLeftHeaderDisposeGameObject_test.re

```diff
@@ -35,8 +35,14 @@ let _ =
               (),
             ),
           (),
-        )
-      );
+        );
+
+        LocalStorage.setValue(HeaderNotice.Method.getWelComeUserKey(), "ok");
```

Comment:
why set localStorage?

Reply:
this test use BuildComponentTool.buildHeader func, will build HeaderNotice component